### DISCUSSION TITLE
Is there any way to upload a project "programatically" so that FUXA starts with the selected project config?

### DISCUSSION THREAD

**Discussion by cpinon-grd** [frangoteam/FUXA#722] [Q&A] [answered]
I'm working on a solution to launch multiple containers, one of which is FUXA. I'm wondering if there's a way to automatically load a saved project (JSON file) so that FUXA starts with the designated views instead of having to do it manually. 

Thanks in advance.

**Comment by unocelli** [comment, marked as answer]
Hi,
I am not sure I understand, the project is located in server/_data in a sqlite database but it should also work to upload the project from the webapi POST /api/project
